Raptacon/Robot-2023
# Auto balance

## Types of changes


## Associated Issue(s)
#121

## Steps to test
1. Run in autonomous to see if it reaches the balance board if it will correct itself if necessary to stay balanced. 

## Changelog
Have a different autonomous mode to where breadbox balances during autonomous. 

## Why?
I am making this change so we can preform better during autonomous.
